extensions.tex

```diff
@@ -79,12 +79,44 @@ \subsection{Beta Distribution Bag-of-words}
   \subsection{Grid Searching}
   \label{sec:grid_search}
 
-    \subsubsection{RNN}
-    \label{sec:rnn_grid_search}
+  \subsubsection{RNN}
+  \label{sec:rnn_grid_search}
 
-\subsection{Ensemble}
-\label{sec:ensemble}
+  \subsection{Ensemble}
+  \label{sec:ensemble}
+  
+  \subsection{Translation}
+  \label{sec:translation}
+ To improve the results of some classifiers, the dataset was enhanced by adding translations of the sentences through an intermediate language back to English. This method was shown to work in a previous Kaggle competition, 'Toxic Comments' \cite{Jigsaw2017}. This is a different application however as it categorises based on the sentiment whereas this challenge is based on the stylistic properties of a given author.
+  
+  The thought behind doing this though, is as there are very few sentences currently, we can get more data to perform analysis on. To do this \textit{TextBlob} was used to translate the sentences in to German, Spanish and French and then back from each respective language into English. 
+  
+  Although an issue may be that some style lost in translation, it appears as though there is some enhancement in the loss by using translations of around 0.03 with the Bag-of-words Feature Kernal Neural Net as seen in the Table \ref{tab:bow_nn_res_tran}. The translation may also help prevent the over fitting that is seen without it.
+  
+\begin{table}[h]
+\centering
+\begin{tabular}{m{1cm}|m{1cm}|m{1cm}|m{1cm}|m{0cm}}
+\multicolumn{1}{m{1cm}}{} & \multicolumn{1}{m{1cm}}{EAP} & \multicolumn{1}{m{1cm}}{HPL} & \multicolumn{1}{m{1cm}}{MWS} &\\[5pt]
+\cline{2-4}
+EAP & 0.82561218 & 0.12039488 & 0.05399294 & \\[5pt]
```

Comment:
Can we round these numbers so that they fit into the table better? Maybe 3 decimal places?

Reply:
I'm going to change them to integers hopefully shortly

Reply:
"The thought behind doing this though" is overly casual.

You need to credit that you pulled this code from an external source in the report (and cite it). APB won't see the codebase itself.

The language in the paragraph on line 94 "although an issue..." is muddled. Needs rewriting.

Reply:
I rewrote some of the paragraph. +1 for the citation though